hysds_commons/stac_utils.py

```diff
@@ -0,0 +1,85 @@
+import copy
+import os
+import json
+from hysds_commons.log_utils import logger
+
+def bbox(coord_list):
```

Comment:
what does this function do?
can you write a docstring for this function?

also can you rename it to something more specific, `bbox` is too generic of a name

Reply:
Is generate_bbox more helpful? The function creates a bbox for an input list of coordinates describing a polygon.  The function is called within the create_stac_doc function. (I will put this in the doctoring)

Reply:
try using pycharm style docstrings instead: https://www.jetbrains.com/help/pycharm/creating-documentation-comments.html#43c544bd

can you explain in detail what the function does? why its sorting the array that way
the for loop is a little hard to understand

Reply:
Ok I will update with the pycharm docstrings 
- The function converts the GeoJSON object `geometry` to a bounding box, or `bbox`. The function takes the parameters of a coordinate list `coord_list`, iterates through the list and returns the final coordinates of the bbox with minimum and maximum values for 4 different points. The field enables more naive clients to easily index and search geospatially

Reply:
@kynjonez: if this is addressed, please click on "Resolve converstation".